server-worker/src/main/java/com/aliyun/emr/rss/service/deploy/worker/LocalFileMeta.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.aliyun.emr.rss.service.deploy.worker;
+
+import java.io.File;
+import java.util.ArrayList;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LocalFileMeta {
+  String shuffleKey;
+  String fileName;
+  File file;
+  long bytesFlushed = 0;
+  ArrayList<Long> chunkOffsets = new ArrayList<>();
+
+  @JsonCreator
```

Comment:
In my option the JSON format is not efficient enough, I think protobuf would be a superior choice.

Reply:
> In my option the JSON format is not efficient enough, I think protobuf would be a superior choice.

Yea, so let me remove this json related code in this pr, add add protocol ser/de in another patch(support recover from level db)?